### PR TITLE
Fix partition overwrite assertion

### PR DESCRIPTION
**Pull request by aczajkowski**
Test was missing final check that all old records where removed.

### CODE REVIEW

File: plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveInsertOverwrite.java [hunk 1]
```diff
@@ -199,12 +199,14 @@ protected void assertOverwritePartition(String testTable)
...
 
-        query(format("INSERT INTO %s values('POLAND', 'Overwrite', 25, 5)", testTable))
+        computeActual(format("INSERT INTO %s values('POLAND', 'Overwrite', 25, 5)", testTable));
+        query(format("SELECT count(*) FROM %s WHERE regionkey = 5", testTable))
                 .assertThat()
                 .skippingTypesCheck()
                 .containsAll(resultBuilder(getSession())
                         .row(1L)
                         .build());
+
```
Comment (wendigo) [inline plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveInsertOverwrite.java:209]: remove this line

**Review by wendigo — APPROVED** [review]

**Comment by aczajkowski** [comment]
@wendigo clicked by accident re-review. Sorry for that. Please re-approve if there are no any other issues.

**Review by losipiuk — APPROVED** [review]

File: plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveInsertOverwrite.java [hunk 2]
```diff
@@ -199,7 +199,8 @@ protected void assertOverwritePartition(String testTable)
                         .row(2L)
                         .build());
 
-        query(format("INSERT INTO %s values('POLAND', 'Overwrite', 25, 5)", testTable))
+        computeActual(format("INSERT INTO %s values('POLAND', 'Overwrite', 25, 5)", testTable));
+        query(format("SELECT count(*) FROM %s WHERE regionkey = 5", testTable))
```
Comment (losipiuk) [inline plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveInsertOverwrite.java:203]: That is ok - though I would prefer asserting on actual rows, not count.

Comment (aczajkowski) [reply plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveInsertOverwrite.java]: Added second commit to switch to exact rows.

**Pull request merged**